components/ArticlesList.jsx

```diff
@@ -0,0 +1,44 @@
+import ArticleCard from "./ArticleCard";
+import { useState, useEffect } from "react";
+
+function ArticlesList() {
+
+    const [articlesArray, setArticlesArray] = useState([]);
+
+    useEffect(() => {
+        async function fetchArticles() {
+            try {
+                const response = await fetch(`https://first-nc-project.onrender.com/api/articles`);
+                console.log(response, 'bye');
+                const data = await response.json();
+                setArticlesArray(data.articles);
+            } catch (error) {
+                // handle error
+            }
+        }
+        fetchArticles();
+    }, []);
+
+    function articlesCardMap() {
+        return articlesArray.map(article => {
```

Comment:
we could do this functionality in the articleCard component eby passing down the articlesArray